Documentation/ApiOverview/Backend/ContextualMenu.rst

```diff
@@ -31,15 +22,18 @@ Context menu rendering flow
 Markup
 ------
 
-The context menu is shown after click on the HTML element which has
+The context menu is shown after clicking on the HTML element which has
 :html:`class="t3js-contextmenutrigger"` together with :html:`data-table`,
 :html:`data-uid` and optional :html:`data-context` attributes.
 
 The JavaScript click event handler is implemented in the
-:js:`TYPO3/CMS/Backend/ContextMenu` requireJS module. It takes the
-data attributes mentioned above and executes an ajax call to the
+ES6 module :js:`@typo3/backend/context-menu.js`. It takes the
+data attributes mentioned above and executes an AJAX call to the
 :php:`\TYPO3\CMS\Backend\Controller\ContextMenuController->getContextMenuAction()`.
 
+..  versionchanged:: 12.0
+    The RequireJS module :js:`TYPO3/CMS/Backend/ContextMenu` has been migrated
```

Comment:
What does that mean to me? What have I to do? Can we link to the tutorial section below (although it is not prepared for ES6 yet)?

Reply:
added link